src/components/AgreementSections/FormInfo.js

```diff
@@ -67,6 +73,13 @@ export default class FormInfo extends React.Component {
     return null;
   }
 
+  validateUniqueProperty = (value, _, meta) => {
+    const { checkAsyncValidation, values = {} } = this.props;
+    if (!value || value === values[meta.name]) return '';
```

Comment:
What's the second part of the OR here doing? "If the currently entered value is equal to...the currently entered value"? Are `values` (which, sidenote, isn't that what the `_` arg is?) the values that already passed validation and are part of the form state?

Reply:
Hmm `values`  actually needs to be `initialValues` here. So that the form lets us submit it even though we edit another field on it. Will change it